docs/proposals/job.md

```diff
@@ -0,0 +1,135 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pod(s) that require to run on demand.
```

Comment:
With "on demand" I've meant that you need to explicitly create those, but I doesn't make sense that way. I'll rewrite that to talk about run-once pods.